CHANGELOG.md

```diff
@@ -12,11 +12,15 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
-## v1.5.1 (WIP)
+## 1.5.1 (WIP)
 
 - Fixed version panel misplacement on scrollable pages and being locked to the
   width of the side nav bar, a bug introduced in v1.5.0 by #82. (#97)
 
+- Adds the ability to login. While logged in this will forward the OIDC
+  access token to the backend such that a secure user access control is
+  established. (#70)
+
```

Comment:
Need to merge/rebase in latest master as v1.5.1 (#99) was just released.

Suggest using the version v1.6.0, as I think this is a minor-bump worthy change

Reply:
Fixed.